src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
+/* eslint-disable no-unused-vars */
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
```

Comment:
next time, would be better to resolve message from promise, and handle it 
promise1.then().catch()